source/lib.ts

```diff
@@ -126,8 +131,8 @@ const parseOptions = (
 			_response: Response,
 			_optionsUsed: Options,
 		): void => {},
-		// Allow the above to be overriden by the options passed to the middleware
-		...passedOptions,
+		// Allow the options object to be overrode by the options passed to the middleware.
```

Comment:
```suggestion
		// Allow the default values to be overriden by the options passed to the middleware.
```

I think overriden (the past participle) fits here